iptables/tables.sls

```diff
@@ -4,15 +4,21 @@
 {%- from "iptables/map.jinja" import firewall with context %}
 
 {%- for t in ['filter','nat','mangle'] %}
-  {%- for cn, cv in firewall.get(t)|dictsort %}
+  {%- for cn, cv in firewall.get(t).items() %}
     {%- set pol = cv.policy | default('ACCEPT') %}
     {%- set rules = cv.rules | default({}) %}
```

Comment:
@mgomersbach , the way you split the loop in two loops renders [these two lines above](https://github.com/saltstack-formulas/iptables-formula/pull/48/files#diff-900a92f9dfe3ca867e6697161bb736baR8-R9) useless in the chain-creation loop, so they can be removed too.

Reply:
Will remove those